### PR TITLE
Update README.md 

### PR DESCRIPTION
**Pull request by ErnestPH**
Updating the instruction on how to update AzuraCast's docker… 

This is to skip of having to answer the question, "Update docker-compose.yml file? This will overwrite any customizations you made to this file? [y/N]" - As we always type "y".

### CODE REVIEW

File: docs/install/README.md [hunk 1]
```diff
@@ -80,15 +80,14 @@ Don't worry if you aren't sure of these items yet; you can always make changes t
 Using the included Docker utility script, updating is as simple as running:
 
 ```bash
-./docker.sh update-self
-./docker.sh update
+./docker.sh update-self && echo "y" | ./docker.sh update
```
Comment (Vaalyn) [inline docs/install/README.md]: I'd probably not just replace the existing "manual" command but add this as a second more "automatic" way to update.

Something like this:

```bash
./docker.sh update-self && ./docker.sh update
```
```
Using the following command to update will automatically answer yes to the question about overwriting the docker-compose.yml:
```

```bash
./docker.sh update-self && echo "y" | ./docker.sh update
```

Comment (ErnestPH) [reply docs/install/README.md]: Of course, you are correct. thanks! I will add it again.

**Comment by SlvrEagle23** [comment]
I don't want the main readme on the GitHub to have this, as it will likely confuse the casual reader looking to update their installation, and this prompt is intended for people who are automatically updating. I'll add it to the main site documentation.

**Comment by ErnestPH** [comment]
Ok understood. Thanks!

**Comment by SlvrEagle23** [comment]
@ErnestPH Disregard my earlier deleted message; I thought we were in the AzuraCast/AzuraCast repository.

**Pull request merged**